doc/sphinx/source/recipes/recipe_hydrology.rst

```diff
@@ -43,13 +77,40 @@ Variables
    * tas (atmos, daily, longitude, latitude, time)
    * pr (atmos, daily, longitude, latitude, time)
 
+
+#. recipe_marrmot.yml
+
+   * tas ( longitude, latitude, time)
+   * pr (longitude, latitude, time)
+
+  MARRMoT requires potential evapotranspiration (PET). Thus, it might also need psl, rsds, rsdt to be used in debruin function (De Bruin et al. 2016).
```

Comment:
From the code it looks like the diagnostic script will crash if these variables are not there and providing `evspsblpot` instead does not work. I think that either this should be changed so it works, or the documentation should be made more clear.

Reply:
Thanks. Done.